.docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:bionic AS build
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends wget meson debhelper git gcc g++ intltool pkg-config glib2.0 libxml2-dev unzip checkinstall && \
+    apt-get upgrade -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN wget http://ftp.acc.umu.se/pub/GNOME/sources/aravis/0.6/aravis-0.6.4.tar.xz && \
```

Comment:
I would think about using variables to parametrize the particular versions of the libraries.

Reply:
Sure, but I think that will be in a future iteration then as I have to check if the libraries/download locations follow a fixed scheme.

Reply:
Off-topic perhaps, but what is the relationship with [AravisProject/aravis](https://github.com/AravisProject/aravis)?

Reply:
> Off-topic perhaps, but what is the relationship with [AravisProject/aravis](https://github.com/AravisProject/aravis)?

It is the same repository/library. The URL used in the `Dockerfile` is what they refer to in the `README.md` for downloads.

Reply:
It is just a general recommendation for future projects, which is optional, since I've approved everything.